packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -203,6 +216,48 @@ export interface EngineReportingOptions<TContext> {
    * TODO(helen): LINK TO EXAMPLE FUNCTION? e.g. a function recursively search for keys to be blocklisted
    */
   sendVariableValues?: VariableValueOptions;
+  /**
+   * This option allows configuring the behavior of request tracing and
+   * reporting to [Apollo Graph Manager](https://engine.apollographql.com/).
+   *
+   * Note: APOLLO_KEY must be set to send traces to Apollo Graph Manager. If it is
```

Comment:
OK this isn't actually true because there are other ways to set the API key.
I guess my main point was just that it's a little odd to say to say all traces are reported by default when you actually still have to enable reporting in the first place. Maybe just remove this paragraph, never mind.